package protoFile.hall.ReadyForQueueRoom;
message ForQueueRoomReques{//组队房间准备与取消准备
     required int32 id = 1;		//角色的ID
     required int32 readyStatu = 2;	//要更新的准备状态 0 取消准备 1准备
}

message ForQueueRoomResponse{
     required bool result = 1;
     optional string message = 2;
}